api/app/models/annotation.rb

```diff
@@ -146,4 +160,11 @@ def public?
     !private
   end
 
+  private
+
+  def trigger_event_creation
+    return if %w(highlight resource).include? format
```

Comment:
These should not be string literals. You should instead be referencing the already defined constants. I've added a FIXUP commit that addresses this.